data/adapters/simple-treasuries.ts

```diff
@@ -208,7 +208,12 @@ const orgs: Org[] = [
   },
   {
     id: 'olympus',
-    addresses: ['0x31F8Cc382c9898b273eff4e0b7626a6987C846E8'],
+    addresses: [
+      '0x31F8Cc382c9898b273eff4e0b7626a6987C846E8', // Olympus Treasury
+      '0x3df5a355457db3a4b5c744b8623a7721bf56df78', // Convex Allocator
+      '0x0316508a1b5abf1CAe42912Dc2C8B9774b682fFC', // Onsen Allocator
```

Comment:
hello ser @dmihal it looks like the current build of open-orgs is missing this Onsen Allocator. It has about $64M of assets in it. 

[Here](https://zapper.fi/bundle/0x31f8cc382c9898b273eff4e0b7626a6987c846e8,0x3df5a355457db3a4b5c744b8623a7721bf56df78,0x0e1177e47151be72e5992e0975000e73ab5fd9d4,0x0316508a1b5abf1CAe42912Dc2C8B9774b682fFC?label=Olympus%20Treasury&id=0xa96bd58b39f08df0ba48b2ffdd0807f5ffe20604) is a bundle created in Zapper that shows the full treasury balance of $875M (as of rn) vs. $810M on open-orgs. 

And [here](https://zapper.fi/account/0x0316508a1b5abf1CAe42912Dc2C8B9774b682fFC) is a filter down to the onsen allocator contract `0x0316508a1b5abf1CAe42912Dc2C8B9774b682fFC` showing the missing $64M.

I haven't been able to build open-orgs locally to be able to determine why that might be.

Reply:
Hey @appleseed-iii, just stumbled upon this comment

The current code for the Olympus treasury is here:
https://cryptostats.community/discover/treasuries/simple-treasuries

Does this include all the necessary contracts?